### ISSUE TITLE
[Suggestion] Holster Selected Tool or Weapon

### ISSUE THREAD

**Issue by DrizztCode** [tsunamayo/Starship-EVO#5168]
Hi - just a quick suggestion - and apologies if this is already possible (and if so please do tell :-))

It would be good to have a key we can press to holster the tool gun or the selected hand weapon, rather than have to leave a toolbar slot empty for the purpose

maybe default to H? (since the holomap is not available in situations where we are holding a weapon or tool)
of maybe 0? (since is the next key on the hotbar)

anyway - apologies again if this is already a control - but i haven't yet found it

thanks again for all the amazing work! :-)

**Comment by Kaiser-Indrasil** [comment]
Agreed, and I think the Tilde key (to the left of 1) would be great for that.

**Comment by BigBadKangaroo** [comment]
Maybe the "^" key could open up a new toolbar, dedicated just for the multitool with all the variants to scrool through?

And this new toolbar could be rearanged by hand just like setting up a toolbar right now so we can have only the tools we want/need in this? or even just in the position we want (favorites come first etc)

**Comment by DrizztCode** [comment]
> Maybe the "^" key could open up a new toolbar, dedicated just for the multitool with all the variants to scrool through?
> 
> And this new toolbar could be rearanged by hand just like setting up a toolbar right now so we can have only the tools we want/need in this? or even just in the position we want (favorites come first etc)

that might be a bit much for this suggestion thread lol :-)

i'm cool with just the holster button :-)

so you might want to make a new thread for your extra tool bar idea
(i think just allowing for a few extra regular toolbars would be cool - coz then the player could fill them with more blocks - since they may only use a few tools - but if they want to fill a whole bar with tools they can)

**Comment by ProPeach** [comment]
I'll *always* support suggestions for an extra toolbar, 3 isn't quite enough even with the condensed multiple-bricks-in-one-slot feature. 4 would be great!

**Comment by DrizztCode** [comment]
> I'll _always_ support suggestions for an extra toolbar, 3 isn't quite enough even with the condensed multiple-bricks-in-one-slot feature. 4 would be great!

sure thing - i'm definitely in support of an extra toolbar (although i would not want it to be restricted to just the "tools" as was suggested) - but i definitely think a holster button is good no matter what - and certainly currently so we can save the 1-3 slots that we may currently leave empty for holstering lol :-)

**Comment by tsunamayo** [comment]
This seems to be quite a popular suggestion!
